client/components/theme/index.jsx

```diff
@@ -146,6 +154,7 @@ export class Theme extends Component {
 
 		return (
 			<Card className={ themeClass }>
+				{ this.isBeginnerTheme() && <Ribbon color="green">{ translate( 'BEGINNER' ) }</Ribbon> }
```

Comment:
I think I'd rather `translate( 'Beginner' )` and implement the caps via styling. I'll defer to @mapk tho 😄

Reply:
I like that idea. Let's default the all caps to the CSS.